packages/terra-core-docs/src/terra-dev-site/doc/form-select/example/search/Search.jsx

```diff
@@ -8,7 +8,7 @@ const cx = classNames.bind(styles);
 const SearchSelectExample = () => (
   <>
     <label htmlFor="color-field-1"><strong>Colors</strong></label>
-    <SearchSelect placeholder="Select a color" className={cx('form-select')} inputId="color-field-1">
+    <SearchSelect placeholder="Select a color" ariaLabel="Colors" className={cx('form-select')} inputId="color-field-1">
```

Comment:
this code updates `Colors` in two places, one at input control as `aria-label` prop and another at <span> [added for label values ](https://github.com/cerner/terra-core/blob/42d8b2c7b34548b19153a2250233bccc9826b417/packages/terra-form-select/src/search/Frame.jsx#L702) Since input has got role as `combobox` I think aria-label should be sufficient for all both VO and JAWS . 
I think we can remove the label span and keep the prop `aria-label` on form-search-select.

Reply:
Removed <span> as we are passing aria-label.